docs: update heading in `start.md`

jsparkdev opened the pull request:
Update heading to better match its contents.

Kimbangg reviewed (commented):
@jsparkdev 
It seems that the forked version might have differences from the latest version.
Could you check again after updating it, please?

<img width="701" alt="image" src="https://github.com/user-attachments/assets/80d7e593-94a8-4800-b7cc-2741c71305a7" />

jsparkdev commented:
@Kimbangg 

Thanks for the review!

The [currently deployed version](https://frontend-fundamentals.com/en/code/start.html) looks like this

![homepage](https://github.com/user-attachments/assets/28503c0b-6a8d-48f3-8c18-8e911c0580bb)

And I see that [the version I forked](https://github.com/jsparkdev/frontend-fundamentals) is the latest version.

![upstream](https://github.com/user-attachments/assets/c33a1916-ce47-42c8-9816-18775fb18ce2)

Did I miss something?

Kimbangg reviewed (commented):
It seems that the changes are already in the main branch of the fork. 
I think the branch was created before the update. 
Could you either create a new branch or fork the repository again? :pray:

<img width="340" alt="image" src="https://github.com/user-attachments/assets/15222431-3e45-4c36-8111-631b6e5ef753" />

jsparkdev commented:
@Kimbangg 

Yes I just updated, no change.

Are you looking at the `README.md` file? The file I modified is the `start.md` file.

If this is the case, can I change the contents of the `start.md` file in this PR to match the `README.md`?

Kimbangg reviewed (approved):
LGTM 💯
